List flaky test for Cooldown

In this issue, we will list every flaky test that we disable for now to allow the CI to be green.
Those tests will be fixed during the cool-downs
 ----------------------------------------------- kestra OSS -----------------------------------------------
- PostgresRunnerTest : flowConcurrencyWithForEachItem (most likely on all runners)
- RequestTest: selfSigned (port already taken)
```
io.micronaut.http.server.exceptions.ServerStartupException: Unable to start Micronaut server on *:43560
        at app//io.micronaut.http.server.netty.NettyHttpServer.bind(NettyHttpServer.java:672)
        at app//io.micronaut.http.server.netty.NettyHttpServer.start(NettyHttpServer.java:314)
        at app//io.micronaut.http.server.netty.NettyHttpServer.start(NettyHttpServer.java:124)
        at app//io.kestra.plugin.core.http.RequestTest.selfSigned(RequestTest.java:248)
        at java.base@21.0.8/java.lang.reflect.Method.invoke(Method.java:580)
        at app//io.micronaut.test.extensions.junit5.MicronautJunit5Extension$2.proceed(MicronautJunit5Extension.java:142)
        at app//io.micronaut.test.extensions.AbstractMicronautExtension.interceptEach(AbstractMicronautExtension.java:162)
        at app//io.micronaut.test.extensions.AbstractMicronautExtension.interceptTest(AbstractMicronautExtension.java:119)
        at app//io.micronaut.test.extensions.junit5.MicronautJunit5Extension.interceptTestMethod(MicronautJunit5Extension.java:129)
        at java.base@21.0.8/java.util.ArrayList.forEach(ArrayList.java:1596)
        at java.base@21.0.8/java.util.ArrayList.forEach(ArrayList.java:1596)
    Caused by: java.net.BindException: Address already in use
```
- [AbstractRunnerTest.flowTriggerWithConcurrencyLimit](https://github.com/kestra-io/kestra/pull/11009) (fails a bit especially on Kafka: https://github.com/kestra-io/kestra-ee/actions/runs/17382883576/attempts/1#summary-49344101977)

 ----------------------------------------------- kestra EE -----------------------------------------------

--- a/scheduler/src/test/java/io/kestra/scheduler/SchedulerTriggerChangeTest.java
+++ b/scheduler/src/test/java/io/kestra/scheduler/SchedulerTriggerChangeTest.java
@@ -95,7 +95,9 @@ public class SchedulerTriggerChangeTest extends AbstractSchedulerTest {
 
         // wait for execution
         Flux<Execution> receiveExecutions = TestsUtils.receive(executionQueue, either -> {
-            executionQueueCount.countDown();
+            if (either.getLeft().getFlowId().equals(SchedulerTriggerChangeTest.class.getSimpleName())) {
+                executionQueueCount.countDown();
+            }
         });
 
         // wait for killed
@@ -141,7 +143,7 @@ public class SchedulerTriggerChangeTest extends AbstractSchedulerTest {
             flowQueue.emit(flow);
 
             // wait for the killed
-            assertThat(executionKilledCount.await(1, TimeUnit.MINUTES), is(true));
+            assertThat(executionKilledCount.await(20, TimeUnit.SECONDS), is(true));
             assertThat(((ExecutionKilledTrigger) receiveKilled.blockLast()).getTriggerId(), is("sleep"));
 
             // the trigger is restarted

